### PR TITLE
update EC2 supported instance types

### PR DESCRIPTION
**Pull request by haouc**
*Issue #, if available:*
#308 
*Description of changes:*
Update the limit.go to latest supported EC2 instance types

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by tzneal — APPROVED** [review]
LGTM

File: pkg/aws/vpc/limits.go [hunk 1]
```diff
@@ -2605,36 +2920,6 @@ var Limits = map[string]*VPCLimits{
 		Hypervisor:  "xen",
 		IsBareMetal: false,
 	},
-	"g2.2xlarge": {
```
Comment (sushrk) [inline pkg/aws/vpc/limits.go:2608]: Is this expected to be deleted?

Comment (haouc) [reply pkg/aws/vpc/limits.go:2608]: % aws ec2 describe-instance-types --instance-types g2.2xlarge

An error occurred (InvalidInstanceType) when calling the DescribeInstanceTypes operation: The following supplied instance types do not exist: [g2.2xlarge]

I think this is deprecated type.

Comment (haouc) [reply pkg/aws/vpc/limits.go:2608]: Same error for g2.8xlarge as well

**Review by sushrk — APPROVED** [review]

**Pull request merged**